src/target/at32f43x.c

```diff
@@ -89,7 +89,26 @@ static bool at32f43_mass_erase(target_s *target);
 #define AT32F43x_2K_OB_COUNT 256U
 #define AT32F43x_4K_OB_COUNT 2048U
 
-#define DBGMCU_IDCODE 0xe0042000U
+/*
+ * refman: DEBUG has 5 registers, of which CTRL, APB1_PAUSE, APB2_PAUSE are
+ * "asynchronously reset by POR Reset (not reset by system reset). It can be written by the debugger under reset."
+ * Note that it has no TRACE_IOEN and SWO is controlled by GPIO IOMUX (AF) instead.
+ */
+#define AT32F43x_DBGMCU_BASE       0xe0042000U
+#define AT32F43x_DBGMCU_IDCODE     (AT32F43x_DBGMCU_BASE + 0x00U)
+#define AT32F43x_DBGMCU_CTRL       (AT32F43x_DBGMCU_BASE + 0x40U)
+#define AT32F43x_DBGMCU_APB1_PAUSE (AT32F43x_DBGMCU_BASE + 0x80U)
+#define AT32F43x_DBGMCU_APB2_PAUSE (AT32F43x_DBGMCU_BASE + 0x0cU)
+#define AT32F43x_DBGMCU_SER_ID     (AT32F43x_DBGMCU_BASE + 0x20U)
+
+#define AT32F43x_DBGMCU_CTRL_SLEEP_DEBUG     (1U << 0U)
+#define AT32F43x_DBGMCU_CTRL_DEEPSLEEP_DEBUG (1U << 1U)
+#define AT32F43x_DBGMCU_CTRL_STANDBY_DEBUG   (1U << 2U)
+#define AT32F43x_DBGMCU_CTRL_SLEEP_Msk \
```

Comment:
Please, if you're going to define a constant like this, use `_MASK` not `_Msk`.

Reply:
Renamed.